cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts

```diff
@@ -192,6 +239,83 @@ export default class RealmSettingsPage {
     return this;
   }
 
+  changeTimeUnit(unit: string, inputType: string, listType: string) {
```

Comment:
Since this is TypeScript you can provide a string literal union type here so you cannot accidentally pass in an invalid value.

```suggestion
  changeTimeUnit(unit: "Minutes" | "Hours" | "Days", inputType: string, listType: string) {
```

If you call this method with an invalid value:

```ts
this.changeTimeUnit("Foo", ...)
```

You'll get an error at compile-time:

> Argument of type '"Foo"' is not assignable to parameter of type '"Minutes" | "Hours" | "Days"'